types/react-redux/index.d.ts

```diff
@@ -164,7 +164,7 @@ interface MergeProps<TStateProps, TDispatchProps, TOwnProps, TMergedProps> {
     (stateProps: TStateProps, dispatchProps: TDispatchProps, ownProps: TOwnProps): TMergedProps;
 }
 
-interface Options extends ConnectOptions {
+interface Options<TStateProps = any, TOwnProps = any, TMergedProps = any> extends ConnectOptions {
```

Comment:
@clayne11 @thasner the build was failing because the arguments were required, and next-redux-wrapper imports Options:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/next-redux-wrapper/index.d.ts#L21

I'd prefer we default these for now, fix next-redux-wrapper in a separate PR, then remove the "any" typings.